avplayback.js

```diff
@@ -119,8 +119,13 @@ instance.prototype.actions = function(system) {
 		'play':     { label: 'Play standby clip' },
 		'pause':    { label: 'Pause Resume' },
 		'stop':     { label: 'Stop' },
+		'previewStart':     { label: 'Play preview clip' },
+		'previewPause':    { label: 'Preview Pause' },
+		'previewStop':     { label: 'Preview Stop' },
```

Comment:
indentation of code is not right (not super important, but it is nice if you follow it), since it has been done before you :)